.github/workflows/stable.yml

```diff
@@ -0,0 +1,23 @@
+name: Stable
```

Comment:
@gordon-lim (Can save for future PR) Could you ensure the results of these runs are saved somewhere in an analytics-friendly file format?